packages/integrations-hcl/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@hashicorp/integrations-hcl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
I think we need a build script to output `js` from our ts source. See the [`platform-tools` package.json](https://github.com/hashicorp/web-platform-packages/blob/main/packages/tools/package.json) for an example.